packages/terra-menu/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-menu",
+  "main": "lib/Menu.js",
+  "private": true,
+  "version": "0.1.0",
+  "description": "terra-menu",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-menu",
+    "Menu",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
```

Comment:
This project is currently only scss. We can probably remove the js dependencies.